app/schemas.py

```diff
@@ -22,6 +22,8 @@ class QuestionType(Enum):
     single_choice = "single-choice"
     multi_choice = "multi-choice"
     subjective = "subjective"
+    integer_numerical = "integer-numerical"
```

Comment:
It would be better to use numerical-integer and numerical-float as that indicates that they are a second of categorisation.